### PR TITLE
Update deDE

### PR DESCRIPTION
**Pull request by samyonair**
German Translation

### CODE REVIEW

**Comment by funkydude** [comment]
There's also

```
LegionInvasionTimer.toc
Options\Locales\deDE.lua
Options\LegionInvasionTimer_Options.toc
```

File: Locales/deDE.lua [hunk 1]
```diff
@@ -4,31 +4,31 @@ local _, mod = ...
 local L = mod.L
 
 -- Core
-L.invasionsAvailable = "|T236292:15:15:0:0:64:64:4:60:4:60|t New invasions available!"
-L.tooltipClick = "|cffeda55fClick|r to drag and move."
-L.tooltipClickOptions = "|cffeda55fRight-Click|r to open options."
-L.searching = "Searching..."
+L.invasionsAvailable = "|T236292:15:15:0:0:64:64:4:60:4:60|t Neue Invasionen verfügbar!"
+L.tooltipClick = "|cffeda55fClick|r zum ziehen und bewegen."
+L.tooltipClickOptions = "|cffeda55fRight-Click|r um Optionen zu öffnen."
```
Comment (funkydude) [inline Locales/deDE.lua]: Is `Right-Click` not translated?

**Comment by samyonair** [comment]
I have seen it, will translate further parts tonight

File: Locales/deDE.lua [hunk 2]
```diff
@@ -4,31 +4,31 @@ local _, mod = ...
 local L = mod.L
 
 -- Core
-L.invasionsAvailable = "|T236292:15:15:0:0:64:64:4:60:4:60|t New invasions available!"
-L.tooltipClick = "|cffeda55fClick|r to drag and move."
-L.tooltipClickOptions = "|cffeda55fRight-Click|r to open options."
-L.searching = "Searching..."
+L.invasionsAvailable = "|T236292:15:15:0:0:64:64:4:60:4:60|t Neue Invasionen verf\195\188gbar!"
```
Comment (funkydude) [inline Locales/deDE.lua]: Do not do this. Use proper UTF-8 characters.

Comment (samyonair) [inline Locales/deDE.lua]: Ok no Problem will revert it, have used lua code for it, sorry for that

**Comment by funkydude** [comment]
FYI this is now needing translated: https://github.com/funkydude/LegionInvasionTimer/blob/master/Locales/deDE.lua#L35

**Pull request merged**